go.mod

```diff
@@ -66,6 +66,7 @@ require (
 	github.com/google/go-github/v32 v32.1.0
 	github.com/gopcua/opcua v0.1.12
 	github.com/gorilla/mux v1.6.2
+	github.com/gosnmp/gosnmp v1.28.0
```

Comment:
There is now gosnmp v1.29 out. Do you see any reason to not directly upgrade to the new version?

Reply:
@srebhan, I have version with 1.28.0 tested and running. In my use cases I have not noticed any issues so far which can be related to the upgrade. I see that there is gosnmp v1.29 and that it has only one change included, but I cannot estimate if that change can break the expected behaviors in Telegraf.

If you are willing to go with v1.29.0, I can change it in PR - let me know.

Reply:
Let's try directly going to v1.29 then. Please update the PR.

Reply:
@srebhan, updated to v1.29